JS/scriptGame.js

```diff
@@ -1,21 +1,44 @@
 let images = document.getElementsByTagName("img");
+let toggleButton = document.getElementById("toggleButton");
+let heading = document.querySelector("h1");
+let audioSound = document.getElementById("myAudio");
 window.onload = init;
 function init() {
     let images = document.getElementsByTagName("img");
     for (let i = 0; i < images.length; i++) {
         images[i].onclick = revealImage;
     }
 };
+
 function revealImage(eventObject) {
     let image = eventObject.target; // the target tells what element generated the event
         image.style.filter = "none";
+        audioSound.play();
         setTimeout(brightnessOut, 3000, image);
 }
 
 function brightnessOut(image) {
     let name;
     name = image.style.filter = "brightness(0%)";
-    // for(let i = 0; i < images.length; i++) {
-    //     images[i].style.filter = "brightness(0%)";
-    // }
 }
+function revealImages() {
+    for(let i = 0; i < images.length; i++) {
+        images[i].style.filter = "none";
+    }
+    toggleButton.innerHTML = "Hide Images";
+    heading.style.color="rgb(240, 11, 11)";
```

Comment:
nit: As `rgb(240, 11, 11)` is duplicated in multiple times, you could create a constant for holding the default value